### PR TITLE
Create accessibility statement and link to statement

### PR DESCRIPTION
**Pull request by jonathanglassman**
Created accessibility statement and link as per https://trello.com/c/z3P1iW1H/547-accessibility-statements-and-fixes-for-team-manual-and-tech-docs

### CODE REVIEW

File: source/accessibility.html.md.erb [hunk 1]
```diff
@@ -29,13 +29,13 @@ We know some parts of this website are not fully accessible for the following re
 
 ## Feedback and contact information
 
-If you need any part of this service in a different format like large print, audio recording or braille, email [](mailto:). FIX THIS
+If you need any part of this service in a different format like large print, audio recording or braille, [contact the GOV.UK Account team](https://www.account.publishing.service.gov.uk/feedback?_ga=2.207991701.60345245.1610378262-320443275.1592993989&_gac=1.69290274.1610378262.COuKzsGWlO4CFRMfGwodMBwNQQ).
```
Comment (huwd) [inline source/accessibility.html.md.erb]: Should probaly snip off these GA params!

Comment (huwd) [reply source/accessibility.html.md.erb]:
```suggestion
If you need any part of this service in a different format like large print, audio recording or braille, [contact the GOV.UK Account team](https://www.account.publishing.service.gov.uk/feedback).
```

File: source/accessibility.html.md.erb [hunk 2]
```diff
@@ -29,13 +29,13 @@ We know some parts of this website are not fully accessible for the following re
...
 
 We’ll aim to get back to you within 3 working days.
 
 ## Reporting accessibility problems with this website
 
-We’re always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, email [](mailto:). FIX THIS!
+We’re always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, [contact the GOV.UK Account team](https://www.account.publishing.service.gov.uk/feedback?_ga=2.207991701.60345245.1610378262-320443275.1592993989&_gac=1.69290274.1610378262.COuKzsGWlO4CFRMfGwodMBwNQQ).
```
Comment (huwd) [inline source/accessibility.html.md.erb]: Should probably snip off these GA params

Comment (huwd) [reply source/accessibility.html.md.erb]:
```suggestion
We’re always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, [contact the GOV.UK Account team](https://www.account.publishing.service.gov.uk/feedback).
```

**Review by huwd — CHANGES_REQUESTED** [review]

File: source/accessibility.html.md.erb [hunk 3]
```diff
@@ -17,49 +17,49 @@ This website is run by the GOV.UK Account team at the Government Digital Service
 + navigate most of the website using speech recognition software
 + listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)
 
-We’ve also made the website text as simple as possible to understand.
+We've also written the website in plain English to make it as simple as possible to understand.
```
Comment (huwd) [inline source/accessibility.html.md.erb:20]: do we want to use the `’` characters for apostrophes?

Comment (jonathanglassman) [reply source/accessibility.html.md.erb:20]: I've always used this character in tech content and it hasn't been a problem before. I think it's fine to leave it as is. @huwd can you approve?

**Review by huwd — APPROVED** [review]
